packer/setup.rb

```diff
@@ -187,12 +207,12 @@ def clone_or_update_ruby_by_json(h, work_dir)
   end
 end
 
-Dir.chdir("rails_ruby_bench") do
+Dir.chdir(RAILS_BENCH_DIR) do
   csystem "bash -l -c \"RAILS_ENV=profile ruby seed_db_data.rb\"", "Couldn't seed the database with profiling sample data!"
```

Comment:
The "-l" is in order to run the rvm setup. Non-login shells don't load rvm at all. Without the -l, you need to make sure that whatever Ruby you ran setup.rb with originally has all the current Discourse gems installed into -- this might be a separate operation from installing them in any of the Rubies it currently does.